apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -87,11 +89,12 @@ $(document).ready(() => {
     }
 
     alreadySubmitted = true;
-    // Clean up school data and set age for teachers.
+    // Clean up school data, log if school was not specified, and set age for teachers.
```

Comment:
For my understanding: where is it determined that the school is not specified here?

Reply:
Oh thank you I missed this! This was for a solution I didn't go with for another event.